src/notification/CommandNotification.js

```diff
@@ -0,0 +1,35 @@
+import { RichEmbed, Message} from "discord.js";
+import GrouperClient from "../structures/GrouperClient";
+import Notification from './Notification.js'
+
+export default class CommandNotification extends Notification {
+    /**
+     * Creates a new notification handler for when a command is executed
+     * 
+     * @param {GrouperClient} client 
+     * @param {String} guild - Support server snowflake
+     * @param {String} channel - Alert channel snowflake
+     */
+    constructor(client, guild, channel) {
+        super(client, guild, channel);
```

Comment:
We can reference the alert channel from the client properties, no need to pass it again.